client/src/components/application/NonlinearStepper/ProInfos.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import { Select, Container, FormControl } from '@material-ui/core';
+import MenuItem from '@material-ui/core/MenuItem';
+import InputLabel from '@material-ui/core/InputLabel';
+import PropTypes from 'prop-types';
+
+const ProInfos = ({ handleFormInput }) => (
+  <>
+    <Container maxWidth="sm">
+      <FormControl variant="outlined" fullWidth>
+        <InputLabel htmlFor="outlined-age-native-simple">
+          How confident are you at understanding English ?
+        </InputLabel>
+        <Select
+          fullWidth
+          labelId="english-understand"
+          label="How confident are you at understanding English ?"
+          name="eConfident"
+          onChange={handleFormInput}
+        >
+          <MenuItem value={10}>Ten</MenuItem>
+          <MenuItem value={20}>Twenty</MenuItem>
+          <MenuItem value={30}>Thirty</MenuItem>
+        </Select>
+        <br />
+      </FormControl>
+      <FormControl variant="outlined" fullWidth>
+        <InputLabel id="english-understand">
+          How confident are you at speaking English ?
+        </InputLabel>
+        <Select
+          fullWidth
+          label="How confident are you at speaking English"
+          labelId="demo-simple-select-label"
+          id="demo-simple-select"
+          name="eUnderstand"
+          onChange={handleFormInput}
+        >
+          <MenuItem value={10}>Ten</MenuItem>
+          <MenuItem value={20}>Twenty</MenuItem>
+          <MenuItem value={30}>Thirty</MenuItem>
+        </Select>
+        <br />
+      </FormControl>
+      <FormControl variant="outlined" fullWidth>
+        <InputLabel id="employment-status">
+          What is current Employment status ?
+        </InputLabel>
+        <Select
+          fullWidth
+          label="What is current Employment status ?"
+          labelId="demo-simple-select-label"
+          id="demo-simple-select"
+          name="currentEmploy"
+          onChange={handleFormInput}
+        >
+          <MenuItem value={10}>Ten</MenuItem>
+          <MenuItem value={20}>Twenty</MenuItem>
+          <MenuItem value={30}>Thirty</MenuItem>
+        </Select>
+        <br />
+      </FormControl>
+      <FormControl variant="outlined" fullWidth>
```

Comment:
@alidah0 This FormControl pattern is repeated a few times in this file, so you could refactor it into a small helper component and save yourself some writing / copying if you want.  It could look something like this:
```
const FormQuestion = ({label, name, values, handleFormInput}) => ( //values is an array of { value, text }
  <FormControl variant="outlined" fullWidth>
        <InputLabel id={`${name}-label`}>
          {label}
        </InputLabel>
        <Select
          fullWidth
          label={label}
          labelId="demo-simple-select-label"
          id={`${name}-select`}
          name={name}
          onChange={handleFormInput}
        >
          {values.map(({value, text}) => <MenuItem value={value}>{text></MenuItem>)}
        </Select>
        <br />
      </FormControl>
)
```
And then you could use it in the component like this:
```
<FormQuestion
  label="What is your current employment status?"
  name="currentEmploy"
  values={[{value: 'notEmployed', text: 'I am not employed'}, {value: 'employed', text: 'I am currently employed'}]}
  handleFormInput={handleFormInput}
/>
```
It's not a required or urgent change but it might save some typing and make the file shorter.  Also, by removing some of the duplication you protect yourself from having typos when copying and pasting different ones, or fixing a bug in one but forgetting to fix it in all of the others.